docs/CHANGELOG.rst

```diff
@@ -68,6 +68,8 @@ Changed
 * Add environment export for Jenkins so that the manager will use a
   globally-unique channel name
 * Require Resolwe 7.x
+* Speed-up management command for inserting mappings
+* Processor ``archive-samples`` to create different IGV session files for different ``build`` and ``species``
```

Comment:
Prepend "Add". Maybe you can also leave out the first occurance of word "different".